src/renderer/features/governance/Locks/model/locks-model.ts

```diff
@@ -0,0 +1,77 @@
+import { createStore, createEffect, sample, createEvent, restore, combine } from 'effector';
+import { ApiPromise } from '@polkadot/api';
+import { BN, BN_ZERO } from '@polkadot/util';
+import { spread } from 'patronum';
+
+import type { ChainId, TrackId } from '@shared/core';
+import { governanceService } from '@shared/api/governance';
+import { networkModel } from '@entities/network';
+
+const chainIdChanged = createEvent<ChainId>();
+
+const $maxLock = createStore<BN>(BN_ZERO);
+const $chainId = restore(chainIdChanged, null);
+const $isLocksRequested = createStore(false).reset(chainIdChanged);
+const $isLoading = createStore(true).reset(chainIdChanged);
+
+const getClassLocksFx = createEffect((api: ApiPromise): Promise<Record<TrackId, BN>> => {
+  // return governanceService.getTrackLocks(api, '12mP4sjCfKbDyMRAEyLpkeHeoYtS5USY4x34n9NMwQrcEyoh');
```

Comment:
Do you need this commented code? Maybe create variable for this address?

Reply:
Testing address for POC, will be removed as soon as I will start working on Locks task